okonomiyaki/file_formats/_egg_info.py

```diff
@@ -1015,6 +1054,28 @@ def dump(self, path):
             if self.pkg_info:
                 self.pkg_info._dump_as_zip(zp)
 
+    def to_json_dict(self):
+        if self.platform is None:
+            epd_platform = None
+        else:
+            epd_platform = six.text_type(self.platform)
+
+        return {
+            _JSON_METADATA_VERSION: six.text_type(self.metadata_version),
+            _JSON__RAW_NAME: self._raw_name,
+            _JSON_NAME: self.name,
```

Comment:
I see. Although, that still doesn't answer my question really. Is it something we need or not? The comment there suggests not and here it clearly isn't used at all anyway.